### ISSUE TITLE
App says "failed to take capture" but video plays fine

### ISSUE THREAD

**Issue by newadventure079** [theophanemayaud/video-simili-duplicate-cleaner#109]
I get the following error often for a various number of videos.  
`[17:23:28] ERROR with video.wmv : failed to take capture`

I played the file in VLC, IINA, Movist Pro and it played fine.  I even checked for corruption with ffprobe.

```
bash-3.2$ ffprobe 'video.wmv'; echo $?
ffprobe version 5.1.2 Copyright (c) 2007-2022 the FFmpeg developers
  built with Apple clang version 14.0.0 (clang-1400.0.29.202)
  configuration: --prefix=/opt/homebrew/Cellar/ffmpeg/5.1.2_6 --enable-shared --enable-pthreads --enable-version3 --cc=clang --host-cflags= --host-ldflags= --enable-ffplay --enable-gnutls --enable-gpl --enable-libaom --enable-libaribb24 --enable-libbluray --enable-libdav1d --enable-libmp3lame --enable-libopus --enable-librav1e --enable-librist --enable-librubberband --enable-libsnappy --enable-libsrt --enable-libsvtav1 --enable-libtesseract --enable-libtheora --enable-libvidstab --enable-libvmaf --enable-libvorbis --enable-libvpx --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxml2 --enable-libxvid --enable-lzma --enable-libfontconfig --enable-libfreetype --enable-frei0r --enable-libass --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-libspeex --enable-libsoxr --enable-libzmq --enable-libzimg --disable-libjack --disable-indev=jack --enable-videotoolbox --enable-neon
  libavutil      57. 28.100 / 57. 28.100
  libavcodec     59. 37.100 / 59. 37.100
  libavformat    59. 27.100 / 59. 27.100
  libavdevice    59.  7.100 / 59.  7.100
  libavfilter     8. 44.100 /  8. 44.100
  libswscale      6.  7.100 /  6.  7.100
  libswresample   4.  7.100 /  4.  7.100
  libpostproc    56.  6.100 / 56.  6.100
Input #0, asf, from 'video.wmv':
  Metadata:
    title           : title
    copyright       : copyright
    WM/WMADRCPeakReference: 32767
    WM/WMADRCAverageReference: 3605
    Buffer Average  : 39506
    WMFSDKVersion   : 10.00.00.3646
    WMFSDKNeeded    : 0.0.0.0000
    VBR Peak        : 1600000
    IsVBR           : 1
  Duration: 00:04:34.49, start: 0.235000, bitrate: 688 kb/s
  Stream #0:0(eng): Audio: wmav2 (a[1][0][0] / 0x0161), 22050 Hz, 2 channels, fltp, 22 kb/s
  Stream #0:1(eng): Video: wmv3 (Main) (WMV3 / 0x33564D57), yuv420p, 320x240, 700 kb/s, 29.97 fps, 29.97 tbr, 1k tbn
0 <---success
```
ffprobe result of `1` is failure and `0` is success

What command is the app using to detect corrupt videos?  Is it using ffmpeg/ffprobe version 5?

**Comment by begijnhofdendermonde** [comment]
Many of my videos are encoded with AC1 codec and all these give this error, it seems the app will have to be updated to also use that codec

**Comment by theophanemayaud** [comment]
Hey @begijnhofdendermonde thanks for contributing! I'm not sure what you mean by AC1, it only seems to be an audio codec... Did you mean AV1 ?

**Comment by daddie888** [comment]
Sorry, I meant AV1 which produces videos that are much smaller with the same quality
Here ascreenshot in Handbrake
![image](https://github.com/user-attachments/assets/d04d2eca-5dd2-4e82-9504-cfa014f7e92a)

**Comment by theophanemayaud** [comment]
Hey, sorry for the delay and thanks for clarifying ! So after two full days of work I finally got av1 to work. The problem is that ffmpeg natively doesn't support av1 for some reason, but two popular libraries can be used to make ffmpeg compatible with av1. I had not yet added any libraries to ffmpeg (which itself is a library of the app) so it took some work, but I should be able to publish a new version in the next week or so with av1 support !!!
In the meantime another new version with logs and saved prefs is coming out, so I'll wait a bit until I see if there are bug reports, before pushing another update with av1 support. I'll inform you here as soon as it's coming out !

**Comment by theophanemayaud** [comment]
Hi @daddie888 the update just went out with av1 support, v1.12.0 ! Hope it helps you !!!

**Comment by suicidejesus666** [comment]
> Hi @daddie888 the update just went out with av1 support, v1.12.0 ! Hope it helps you !!!

Is the Windows version also updated with AV1 support or only the MacOS version for now?

**Comment by theophanemayaud** [comment]
macOS only for now. Unfortunately I’ve been a bit stuck on windows because of arm, I work on a MacBook Pro M series. QT only released this fall compatibility with arm, and I’ve just started to rework the windows version to be compatible with arm toolchains. I expect it to still take me a few months, but I still plan to update windows asap !

**Comment by theophanemayaud** [comment]
@newadventure079 

> This issue was actually for x264 videos and not av1 videos, but I will test again and see what the logs say now

Yes multiple things were mixed in this GitHub issue. I know there are also other files where it fails to take captures and I have to investigate each case a bit specifically. If you have videos where that happens and are ok with sending them over, it’d help figuring out what edge case they fall in and addressing it !

**Comment by daddie888** [comment]
Too bad, I'm on Windows, will have to wait longer ..
